instaBrute.py

```diff
@@ -3,12 +3,16 @@
 from time import sleep
 from selenium import webdriver
 from selenium.webdriver.common.keys import Keys
+import os
 import simplejson as json
 import sys
 import optparse
 
-profile = webdriver.FirefoxProfile()
-profile.set_preference("general.useragent.override", "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/29.0.1547.76 Safari/537.36")
+#profile = webdriver.FirefoxProfile()
+#profile.set_preference("general.useragent.override", "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/29.0.1547.76 Safari/537.36")
+chromedriver = "./chromedriver"
```

Comment:
You're assuming everyone will have the ChromeDriver path. And then setting it as a environment variable. 

I would suggest 2 ways.
1) Delete these two lines and the user is responsible for setting up the environment variables beforehand.
2) Add the path of the binary as a argument